### ISSUE TITLE
[management] [gateway] upgrade gravitee-common

### ISSUE THREAD

**Issue by leleueri** [gravitee-io/issues#7115]
## :collision: Describe the bug
Since the upgrade to gravitee-node 1.20, we should consider upgrade also gravitee-common to avoid missing Field during stop 
```

13:43:45.027 [JettyShutdownThread] [] INFO  o.e.j.s.h.ContextHandler.management - Closing Spring root WebApplicationContext
13:43:45.029 [JettyShutdownThread] [] INFO  o.e.j.server.handler.ContextHandler - Stopped o.e.j.s.ServletContextHandler@6f70015{/management,null,STOPPED}
Exception in thread "graviteeio-finalizer" java.lang.NoSuchFieldError: STOPPING
	at io.gravitee.node.container.AbstractNode.preStopComponents(AbstractNode.java:242)
	at io.gravitee.node.container.AbstractNode.doStop(AbstractNode.java:138)
	at io.gravitee.common.component.AbstractLifecycleComponent.stop(AbstractLifecycleComponent.java:41)
	at io.gravitee.node.container.spring.SpringBasedContainer.doStop(SpringBasedContainer.java:94)
	at io.gravitee.common.component.AbstractLifecycleComponent.stop(AbstractLifecycleComponent.java:41)
	at io.gravitee.node.container.AbstractContainer$ContainerShutdownHook.run(AbstractContainer.java:161)
```

**Comment by tcompiegne** [comment]
@leleueri I cannot reproduce it

**Comment by leleueri** [comment]
Yes sorry, I shouldn't open this issues. It wasn't impact this version (only the ongoing 3.16) but it has been fixed into lastest 3.14.
